### ISSUE TITLE
Unexpected token: deprecate_msg

### ISSUE THREAD

**Issue by peepo5** [fabianboesiger/ftx#72]
When updating to newer version, out of the blue it gave me this:
```rs
> $ cargo build                                                                                                                                                                                       [±release ●●]
    Updating git repository `https://github.com/fabianboesiger/ftx`
   Compiling ftx v0.4.0 (https://github.com/fabianboesiger/ftx?branch=main#1bf760f0)
error: unexpected token: `deprecate_msg`
   --> /home/admin/.cargo/git/checkouts/ftx-9fce05f641d55c68/1bf760f/src/rest/mod.rs:174:18
    |
35  | impl Rest {
    |           - while parsing this item list starting here
...
174 |     #[deprecated=deprecate_msg!()]
    |                  ^^^^^^^^^^^^^
...
534 | }
    | - the item list ends here

error: aborting due to previous error

error: could not compile `ftx`
```

I am using:
Ubuntu 20.04 based os
rust edition = "2018"
and to get module I am using:
```toml
ftx = { git="https://github.com/fabianboesiger/ftx", branch = "main" }
```

**Comment by peepopoggers** [comment]
Alright I think I found the issue, even after installing a newer rustc version it did not switch from old to new so that seems to be the problem. Not sure why is is not switching the rustc version :shrug:

**Comment by peepopoggers** [comment]
I fixed this by doing: `rustup default 1.55.0`
